content/projects/static-site-express.md

```diff
@@ -0,0 +1,15 @@
+---
+title: static-site-express
+repo: SalsaBoy990/static-site-express
+homepage: https://static-site-express.netlify.com/

```

Comment:
This is more of an example than a homepage, should be changed to your GitHub repo.